server/opts.go

```diff
@@ -697,7 +697,21 @@ func (o *Options) processConfigFileLine(k string, v interface{}, errors *[]error
 			}
 			o.TrustedOperators = append(o.TrustedOperators, opc)
 		}
+
+		if o.AccountResolver == nil && len(o.TrustedOperators) == 1 {
```

Comment:
I think we should add a comment here that in case `resolver` is defined that it will take precedence from the value that might be defined in the operator jwt.

Reply:
will add